apps/server/src/shared/domain/entity/materials.entity.ts

```diff
@@ -0,0 +1,68 @@
+import { Entity, Property } from '@mikro-orm/core';
+import { BaseEntityWithTimestamps } from './base.entity';
+
+export interface ITargetGroupProperties {
+	state?: string;
+	schoolType?: string;
+	grade?: string;
+}
+
+export interface IRelatedResourceProperties {
+	originId?: string;
+	relationType?: string;
+}
+
+export interface IMaterialProperties {
+	client: string;
+	description?: string;
+	license: string[];
+	relatedResources: IRelatedResourceProperties[];
+	subjects: string[];
+	tags: string[];
+	targetGroups: ITargetGroupProperties[];
+	title: string;
+	url: string;
```

Comment:
looking at the [old model](https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/src/services/content/material-model.js#L21), I am a bit uneasy that we don't implement some of those fields (originId, contentType, lastModified, language, target, popularity, thumbnailUrl, editorsPick, createdAt). Are these in all cases only in the schema of the old model and actually not used?

Reply:
none of the other types in old materialSchema is in use in any production document objects, checked in dBC and NBC